src/js/Content/Features/Community/ProfileHome/FInGameStoreLink.js

```diff
@@ -1,22 +1,31 @@
-import {HTML, Localization} from "../../../../modulesCore";
-import {Feature, User} from "../../../modulesContent";
+import {GameId, HTML, HTMLParser, Localization} from "../../../../modulesCore";
+import {Feature, RequestData} from "../../../modulesContent";
 
 export default class FInGameStoreLink extends Feature {
 
     checkPrerequisites() {
-        return !this.context.isPrivateProfile && User.isSignedIn;
+        return !this.context.isPrivateProfile
+            && (this._avatarNode = document.querySelector(".profile_header_size[data-miniprofile]")) !== null
+            && this._avatarNode.classList.contains("in-game");
```

Comment:
Is this required for a check on the "report abuse" form?

Reply:
Technically none of these checks are necessary since we're only interested in finding the appid. But in the event the "report abuse" form is removed for good, this check is cheaper to perform for testing if the profile we're viewing is currently in-game or not.